开始训练...
训练集数据处理。。。。
=============
验证集数据处理。。。。。
=============
0
Traceback (most recent call last):
  File "D:/Users/Admin/PycharmProjects/PyCharm_Projects/Deep_learning/cat_dog/model_training02.py", line 116, in <module>
    train(model, EPOCH_NLM, opt)
  File "D:/Users/Admin/PycharmProjects/PyCharm_Projects/Deep_learning/cat_dog/model_training02.py", line 38, in train
    logits = model(img)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\dygraph\layers.py", line 902, in __call__
    outputs = self.forward(*inputs, **kwargs)
  File "D:\Users\Admin\PycharmProjects\PyCharm_Projects\Deep_learning\cat_dog\mnist.py", line 32, in forward
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\dygraph\layers.py", line 902, in __call__
    outputs = self.forward(*inputs, **kwargs)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\nn\layer\common.py", line 129, in forward
    x=input, weight=self.weight, bias=self.bias, name=self.name)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\nn\functional\common.py", line 1451, in linear
    'transpose_Y', False, "alpha", 1)
ValueError: (InvalidArgument) Input X's width should be equal to the Y's height, but received X's shape: [10, 24200], Y's shape: [31250, 64].
  [Hint: Expected mat_dim_x.width_ == mat_dim_y.height_, but received mat_dim_x.width_:24200 != mat_dim_y.height_:31250.] (at C:/home/workspace/Paddle_release/paddle/fluid/operators/matmul_op.cc:594)
  [operator < matmul > error]
